dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/settings/SettingsStore.java

```diff
@@ -103,23 +106,45 @@ public void setFilePropertiesSettings(final boolean settings) {
         filePropertiesSettings.overwrite(settings);
     }
 
+    public ShowCachedJobSettings getShowCachedJobSettings() {
+        return showCachedJobSettings;
+    }
+
+    public void setShowCachedJobSettings(final boolean settings) {
+        dirty = true;
+        showCachedJobSettings.overwrite(settings);
+    }
+
+
     //method to include new json entry to settings.json file if any new setting property is introduced.
     public static void writeNewSettingsToSettingsJsonFile() {
         try (final Stream<String> stream = Files.lines(PATH)) {
-            stream.forEach(e -> {
-                if (!e.contains("filePropertiesSettings")) {
-                    StringBuilder newFile = new StringBuilder(e);
+            stream.forEach(entry -> {
+                if (!entry.contains("filePropertiesSettings")) {
+                    StringBuilder newFile = new StringBuilder(entry);
```

Comment:
I realize that this was previously not final because it's being reassigned later, but I question why it has to be reassigned in the first place.  If the reassignment on line 126 is removed then this can be made final